### PR TITLE
Enabling Disabled Test Cases

### LINKED ISSUE #1448: Investigate and fix the ignored tests and evaluate whether some of them might need to be removed
# Task Description

As seen here:

image

There are currently several test cases which are ignored and need to either be investigated and fixed, or re-evaluated and possibly be removed.

# Tasks

The following tasks will need to be carried out:
* [ ] Fix those tests that make sense to be re-enabled.
* [ ] Drop obsolete tests.

# Useful Links

* [Strongbox: Writing Tests](https://strongbox.github.io/developer-guide/writing-tests.html)
* [Strongbox: JUnit User Guide](https://strongbox.github.io/developer-guide/junit-user-guide.html)
* [Strongbox: Writing Web Form Tests](https://strongbox.github.io/developer-guide/writing-web-form-tests.html)

# Help

* [Our chat](https://chat.carlspring.org/)
* Points of contact:
  * @carlspring
  * @fuss86
  * @sbespalov
  * @steve-todorov

### PR DESCRIPTION
**Pull request by anki2189**
# Pull Request Description

This pull request closes #1448 

# Acceptance Test

* [x] Building the code with `mvn clean install -Dintegration.tests` still works.
* [x] Running `mvn spring-boot:run` in the `strongbox-web-core` still starts up the application correctly.
* [x] Building the code and running the `strongbox-distribution` from a `zip` or `tar.gz` still works.
* [x] The tests in the [`strongbox-web-integration-tests`](https://github.com/strongbox/strongbox-web-integration-tests/) still run properly.

# Questions

* Does this pull request break backward compatibility? 
  * [ ] Yes
  * [x] No

* Does this pull request require other pull requests to be merged first? 
  * [ ] Yes, please see #...
  * [x] No

* Does this require an update of the documentation?
  * [ ] Yes, please see strongbox/strongbox-docs#{PR_NUMBER}
  * [x] No

### CODE REVIEW

File: strongbox-web-core/src/main/java/org/carlspring/strongbox/controllers/users/UserController.java [hunk 1]
```diff
@@ -239,6 +239,11 @@ public ResponseEntity delete(@ApiParam(value = "The name of the user") @PathVari
             return getFailedResponseEntity(HttpStatus.FORBIDDEN, OWN_USER_DELETE_FORBIDDEN, accept);
         }
 
+        if (StringUtils.equals("admin", username))
```
Comment (sbespalov) [inline strongbox-web-core/src/main/java/org/carlspring/strongbox/controllers/users/UserController.java]: @anki2189 why this change needed for?
@carlspring @steve-todorov do we have such requirments to prevent "admin" user been deleted?

Comment (anki2189) [reply strongbox-web-core/src/main/java/org/carlspring/strongbox/controllers/users/UserController.java]: There was one disabled TestCase (`UserControllerTestIT.testDeletingRootAdminShouldBeForbidden`) for deleting admin User . 
According to test case deletion of root admin should not be allowed , so added this code in UserController.

Comment (sbespalov) [reply strongbox-web-core/src/main/java/org/carlspring/strongbox/controllers/users/UserController.java]: ok, thanks for clarifying

**Comment by anki2189** [comment]
> LGTM!
> @carlspring I vote to merge

Thanks @sbespalov .
